server/services/DEPRECATEnotificationService.js

```diff
@@ -3,72 +3,55 @@ const knex = require('knex')
 const db = require('../database')
 
 const DEPRECATEnotificationService = {
-  // this is only used to count the number of notifications in calendar.ejs, and we have two other methods in this class that do the same thing.
+  // this is only used to count the number of notifications in calendar.ejs
   async getShiftNotifications(quantumId) {
     return db
-      .select(
-        knex.raw(
-          `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-        ),
-      )
-      .from('SHIFT_NOTIFICATION')
-      .where('QUANTUM_ID', '=', quantumId.toLowerCase())
+      .select('quantum_id', 'last_modified')
+      .from('shift_notification')
+      .where('quantum_id', '=', quantumId.toLowerCase())
       .union(
         db
-          .select(
-            knex.raw(
-              `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-            ),
-          )
-          .from('SHIFT_TASK_NOTIFICATION')
-          .where('QUANTUM_ID', '=', quantumId.toLowerCase()),
+          .select('quantum_id', 'last_modified')
+          .from('shift_task_notification')
+          .where('quantum_id', '=', quantumId.toLowerCase()),
       )
-      .orderBy('LastModifiedDateTime', 'desc')
       .catch((err) => {
         throw err
       })
   },
 
   getShiftNotificationsPaged(quantumId, offset, perPage) {
     return db
-      .select(
-        knex.raw(
-          `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-        ),
-      )
-      .from('SHIFT_NOTIFICATION')
-      .where('QUANTUM_ID', '=', quantumId.toLowerCase())
+      .select(knex.raw(`description AS "Description", last_modified AS "LastModifiedDateTime"`))
+      .from('shift_notification')
+      .where('quantum_id', '=', quantumId.toLowerCase())
       .union(
         db
-          .select(
-            knex.raw(
-              `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-            ),
-          )
-          .from('SHIFT_TASK_NOTIFICATION')
-          .where('QUANTUM_ID', '=', quantumId.toLowerCase()),
+          .select(knex.raw(`description AS "Description", last_modified AS "LastModifiedDateTime"`))
+          .from('shift_task_notification')
+          .where('quantum_id', '=', quantumId.toLowerCase()),
       )
       .offset(offset)
       .limit(perPage)
-      .orderBy('LastModifiedDateTime', 'desc')
+      .orderBy('LastModifiedDateTime', 'DESC')
       .catch((err) => {
         throw err
       })
   },
 
   updateShiftNotificationsToRead(quantumId) {
```

Comment:
Are we happy to leave the the method name with "Read" in it? Can't remember what the rational for moving from `read` to `processed` was in the db, just wanna check that keeping with this naming works in the new context.

Reply:
I can rename/combine these two methods if you want, but they're not going to be around much longer. your call.

Reply:
Na leave it :)